index.html

```diff
@@ -13,116 +13,129 @@
         <button class="icon">☰</button>
     </header>
     <main>
-        <section class="header">
-            <h1 class="name">Hey there. I'm Juan Sebastian</h1>
-            <h2 class="career">I'm a software developer</h2>
-            <p class="paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</p>
-        </section>
-        <section class="social">
-            <ul>
-                <li><a class="github" href="https://github.com/Juanse7793"><img src="images/github.svg" alt="github"></a></li>
-                <li><a class="linkedin" href="https://www.linkedin.com/in/juan-sebastian-sotomayor-2bb395198/"><img src="images/Linkedin_icon.svg" alt="linkedin"></a></li>
-                <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
-            </ul>
-        </section>
-        <section class="whole-works-section">
-            <section class="works-title">
-                <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
-                <h2 class="recent-works">My Recent Works</h2>
-                <img class="separation-bar" src="Backgroud-images/Indicator.png" alt="separation-bar">
+        <section class="intro">
```

Comment:
Kindly modify this section so that your design resembles Figma's as much as possible 😎 The background is not covering all the width of the screen as you can see 🧐

Your Design 
![image](https://user-images.githubusercontent.com/15807343/154575848-2e6d3987-c4ac-4de1-9eec-1396737b2441.png)

Figma's 
![image](https://user-images.githubusercontent.com/15807343/154575892-8b06ce4f-a77e-44ab-b5ea-a3a1354ae2c1.png)